packages/uswds-core/src/styles/mixins/helpers/checkbox-and-radio-colors.scss

```
@@ -132,14 +138,29 @@
         }
       }
       &:checked:disabled,
-      &:checked[aria-disabled="true"] {
+      &:checked[aria-disabled="true"],
+      &:indeterminate:disabled,
+      &:indeterminate[aria-disabled="true"],
+      &[data-indeterminate]:disabled,
+      &[data-indeterminate][aria-disabled="true"] {
         @include format-input {
           background-color: color("disabled");
           @media (forced-colors: active) {
             background-color: color(GrayText);
           }
         }
       }
+      &:indeterminate,
+      &[data-indeterminate] {
+        &,
+        &:checked {
+          @include format-input {
+            @include add-background-svg("checkbox-indeterminate");
+            background-size: 75%;
+            background-position: center;
```

For consistency, we might want to match how these styles are being set on the `checked` checkboxes
```suggestion
            background-position: center center;
            background-size: units(1.5) auto;
```

For reference:

https://github.com/uswds/uswds/blob/48bfdf2960d7cbd043dac5bf71a9616ec1defa65/packages/usa-checkbox/src/styles/_usa-checkbox.scss#L48-L52